platform/el/run-image.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
I'm not sure what this does - does it run a builder or the final packages etc? A more clear name and docstring is required, especially since `IMAGE_NAME` is required and not specified.